src/chrome/content/rules/Itc.cn.xml

```diff
@@ -48,5 +54,7 @@
 
 	<securecookie host="^\w" name=".+" />
 
+	<rule from="^http://i7\.itc\.cn/" to="https://i0.itc.cn/" />
```

Comment:
I don't get the same content on the provided test url.

Reply:
Oh, it seems like server error.
I'm sure i7 is same to i[0-6]. What should I do?

Reply:
Can you provide other test urls?